### ISSUE TITLE
[Bug] on screen reader moe i cannot see the level of npcs like intended

### ISSUE THREAD

**Issue by jasona** [jasona/darkwind-public#1939]
## Bug Report

on screen reader moe i cannot see the level of npcs like intended

### Context
| Field | Value |
|-------|-------|
| **Submitter** | Serulia |
| **Location** | The Adventurers' Guild |
| **Room Path** | `domains/darkwind/maincity/adv_guild` |
| **Area** | darkwind |
| **Game Version** | 4.5.5 |
| **Client** | unknown |
| **Submitted** | Sat Sep 12 21:04:28 2026 |
| **PG Issue ID** | #0 |
